app/Jobs/Server/UpdateKernelJob.php

```
@@ -0,0 +1,59 @@
+    public function handle(): void
+    {
+        $this->run("server-{$this->server->id}", function () {
+            $this->server->os()->upgradeKernel();
+            $this->server->checkConnection();
+            $this->server->checkForUpdates();
+            $this->broadcastServerUpdate();
+            app(RebootServer::class)->reboot($this->server);
+        });
+    }
+
+    public function failed(Exception $e): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $e. Configured minimum length is 3. (undefined)

(ShortVariable)